ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-v1.6__410.yaml

```diff
@@ -0,0 +1,399 @@
+tests:
+- as: e2e-aws-ocp-410
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.10"
+  steps:
+    test:
+    - as: test
+      cli: latest
+      commands: make test-e2e
+      dependencies:
+      - env: KNATIVE_SERVING_ACTIVATOR
+        name: knative-serving-activator
+      - env: KNATIVE_SERVING_AUTOSCALER
+        name: knative-serving-autoscaler
+      - env: KNATIVE_SERVING_AUTOSCALER_HPA
+        name: knative-serving-autoscaler-hpa
+      - env: KNATIVE_SERVING_CONTROLLER
+        name: knative-serving-controller
+      - env: KNATIVE_SERVING_DOMAIN_MAPPING
+        name: knative-serving-domain-mapping
+      - env: KNATIVE_SERVING_DOMAIN_MAPPING_WEBHOOK
+        name: knative-serving-domain-mapping-webhook
+      - env: KNATIVE_SERVING_STORAGE_VERSION_MIGRATION
+        name: knative-serving-storage-version-migration
+      - env: KNATIVE_SERVING_QUEUE
+        name: knative-serving-queue
+      - env: KNATIVE_SERVING_WEBHOOK
+        name: knative-serving-webhook
+      - env: KNATIVE_SERVING_TEST_AUTOSCALE
+        name: knative-serving-test-autoscale
+      - env: KNATIVE_SERVING_TEST_FAILING
+        name: knative-serving-test-failing
+      - env: KNATIVE_SERVING_TEST_GRPC_PING
+        name: knative-serving-test-grpc-ping
+      - env: KNATIVE_SERVING_TEST_HELLOHTTP2
+        name: knative-serving-test-hellohttp2
+      - env: KNATIVE_SERVING_TEST_HELLOVOLUME
+        name: knative-serving-test-hellovolume
+      - env: KNATIVE_SERVING_TEST_HELLOWORLD
+        name: knative-serving-test-helloworld
+      - env: KNATIVE_SERVING_TEST_HTTPPROXY
+        name: knative-serving-test-httpproxy
+      - env: KNATIVE_SERVING_TEST_PIZZAPLANETV1
+        name: knative-serving-test-pizzaplanetv1
+      - env: KNATIVE_SERVING_TEST_PIZZAPLANETV2
+        name: knative-serving-test-pizzaplanetv2
+      - env: KNATIVE_SERVING_TEST_READINESS
+        name: knative-serving-test-readiness
+      - env: KNATIVE_SERVING_TEST_RUNTIME
+        name: knative-serving-test-runtime
+      - env: KNATIVE_SERVING_TEST_SERVINGCONTAINER
+        name: knative-serving-test-servingcontainer
+      - env: KNATIVE_SERVING_TEST_SIDECARCONTAINER
+        name: knative-serving-test-sidecarcontainer
+      - env: KNATIVE_SERVING_TEST_SINGLETHREADED
+        name: knative-serving-test-singlethreaded
+      - env: KNATIVE_SERVING_TEST_TIMEOUT
+        name: knative-serving-test-timeout
+      - env: KNATIVE_SERVING_TEST_VOLUMES
+        name: knative-serving-test-volumes
+      - env: KNATIVE_SERVING_TEST_WSSERVER
+        name: knative-serving-test-wsserver
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+      timeout: 4h0m0s
+    workflow: generic-claim
+- as: e2e-aws-ocp-410-continuous
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.10"
+  cron: 0 15 * * 1-5
```

Comment:
I suspect we shift this to avoid runs at the same time? 
I see that the job for 1.5 has `0 13 * * 1-5`.

Reply:
Yes, you are correct.
Here is the script https://github.com/openshift/knative-serving/blob/main/openshift/ci-operator/generate-ci-config.sh#L45-L69